python/py-xlrd/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-xlrd
 version             2.0.1
-python.versions     37 38 39 310 311
+python.versions     37 38 39 310 311 312
```

Comment:
Likewise this one's no longer getting updated (hence the conflict with py-excelrd) since python3.9.